examples/01-main-example/sample

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+DMG_FILE_NAME=Application-Installer.dmg
+APP_NAME="Application"
```

Comment:
```suggestion
APP_NAME="Application"
DMG_FILE_NAME="${APP_NAME}-Installer.dmg"
```

The volume name is "`${APP_NAME}-Installer.dmg`", so make the filename match the style.

Reply:
Good point. Also @chunyang-wen since you went ahead you can also add:

```
VOLUME_NAME="${APP_NAME} Installer"

...

--volname "${VOLUME_NAME}" \
```